frontend/types/src/stores/anonymization.d.ts

```
@@ -23,24 +23,12 @@ export interface PatientData {
     };
     id: string;
     status?: string;
+    error?: boolean;
 }
 export declare const useAnonymizationStore: import("pinia").StoreDefinition<"anonymization", AnonymizationState & {
-    pendingAnonymizations: PatientData[];
+    pending: PatientData[];
 }, {}, {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)